src/modules/views/Sharings/index.jsx

```diff
@@ -112,6 +116,11 @@ export const SharingsView = ({ sharedDocumentIds = [] }) => {
         []
       return {
         ...result,
+        // If there are no shared documents, we consider the data is loaded by setting fetchStatus to 'loaded' and lastFetch to now.
+        fetchStatus:
+          sharedDocumentIds?.length > 0 ? result.fetchStatus : 'loaded',
+        lastFetch:
+          sharedDocumentIds?.length > 0 ? result.lastFetch : Date.now(),
```

Comment:
This feels like a hack. do we need to do this? i think isLoaded ( aka sharedDrivesLoaded ) is enough

Reply:
`sharedDrivesLoaded` is not enough, it checks if shared drive list has been loaded or not.
The loading status of shared files/folders need to be checked by `fetchStatus` and `lastFetch`.